<launch>
	<node pkg="lasr_object_detection_yolov8" type="service" name="service" output="screen"/>
	<node pkg="coffee_shop" type="tf_transform.py" name="tf_transform" output="screen"/>
	<node pkg="lasr_shapely" type="service" name="lasr_shapely" output="screen"/>
	<rosparam command="load" file="$(find coffee_shop)/config/test_make_order.yaml" />
	<rosparam command="load" file="$(find coffee_shop)/config/motions.yaml" />
	<node pkg="coffee_shop" type="test_make_order.py" name="test_make_order" output="screen"/>
</launch>